src/MLJText.jl

```diff
@@ -1,115 +1,192 @@
 module MLJText
 
-# The following is just boostrap code to get a working template. You
-# will remove this and replace "import .TextAnalysis" with "import
-# TextAnalysis" and any other deps you need.
-
-module TextAnalysis
-
-function fit(Xmatrix::Matrix, yint::AbstractVector{<:Integer})
-    classes = sort(unique(yint))
-    counts = [count(==(c), yint) for c in classes]
-    Θ = counts / sum(counts)
-end
-
-predict(Xnew::Matrix, Θ) = vcat(fill(Θ', size(Xnew, 1))...)
-
-# julia> yint = rand([1,3,4], 100);
-
-# julia> Θ = fit(rand(100, 3), yint)
-# 3-element Vector{Float64}:
-#  0.35
-#  0.23
-#  0.42
-
-# julia> predict(rand(5, 3), Θ)
-# 5×3 Matrix{Float64}:
-#  0.35  0.23  0.42
-#  0.35  0.23  0.42
-#  0.35  0.23  0.42
-#  0.35  0.23  0.42
-#  0.35  0.23  0.42
-
-end # of module
-
-
-### CONTINUATION OF TEMPLATE
-
-import .TextAnalysis # substitute model-providing package name here (no dot)
+import TextAnalysis # substitute model-providing package name here (no dot)
 import MLJModelInterface
 import ScientificTypesBase
+using SparseArrays, TextAnalysis
 
 const PKG = "TextAnalysis"          # substitute model-providing package name
 const MMI = MLJModelInterface
 const STB = ScientificTypesBase
 
 """
-    CoolProbabilisticClassifier()
-
-A cool classifier that predicts `UnivariateFinite` probability
-distributions. These are distributions for a finite sample space whose
-elements are *labeled*.
+    TfidfTransformer()
```

Comment:
And the `struct` as CameCase like julia convetions:
```suggestion
    TfIdfTransformer()
```

Reply:
this probably needs to be changed downstream also

Reply:
this follows sklearn's representation of the object/class/struct in camel case - I think it's ok for now?  Don't mind changing either though.

Reply:
We better follow the sklearn guidelines. My bad